playground/backend/cmd/server/controller.go

```diff
@@ -184,6 +193,23 @@ func (controller *playgroundController) GetPrecompiledObjectOutput(ctx context.C
 	return &response, nil
 }
 
+// PutPrecompiledObjectsToCategory adds categories with precompiled objects to protobuf object
+func PutPrecompiledObjectsToCategory(categoryName string, precompiledObjects *cloud_bucket.PrecompiledObjects, sdkCategory *pb.Categories) {
```

Comment:
Can we put it into some utils package? As it is done in [Aydar's PR](https://github.com/akvelon/beam/pull/72) with refactoring to keep only controller methods in controller.go?